scripts/make_glibc_and_sysroot.sh

```diff
@@ -124,6 +124,16 @@ $CC $CFLAGS $WARNINGS $EXTRA_FLAGS \
     -MD -MP -MF $GLIBC/build/nptl/elision-unlock.o.dt \
     -MT $GLIBC/build/nptl/elision-unlock.o
 
+$CC $CFLAGS $WARNINGS $EXTRA_FLAGS \
```

Comment:
This looks very similar to the blocks around it.  It's not a problem, but perhaps we should abstract this in a different PR to make it easier to add more in the future without just adding more of the same